src/utils/clack-utils.ts

```diff
@@ -1,32 +1,27 @@
 // @ts-ignore - clack is ESM and TS complains about that. It works though
 import * as clack from '@clack/prompts';
+import * as Sentry from '@sentry/node';
 import axios from 'axios';
 import chalk from 'chalk';
-import * as childProcess from 'child_process';
-import * as fs from 'fs';
-import * as path from 'path';
-import * as os from 'os';
+import * as childProcess from 'node:child_process';
+import * as fs from 'node:fs';
+import * as os from 'node:os';
+import { basename, dirname, isAbsolute, join, relative } from 'node:path';
+import { URL } from 'node:url';
+import opn from 'opn';
 import { setInterval } from 'timers';
-import { URL } from 'url';
-import * as Sentry from '@sentry/node';
-import { hasPackageInstalled, PackageDotJson } from './package-json';
-import { Feature, SentryProjectData, WizardOptions } from './types';
 import { traceStep } from '../telemetry';
+import { debug } from './debug';
+import { hasPackageInstalled, PackageDotJson } from './package-json';
 import {
   detectPackageManger,
   PackageManager,
   packageManagers,
 } from './package-manager';
-import { debug } from './debug';
 import { fulfillsVersionRange } from './semver';
+import { Feature, SentryProjectData, WizardOptions } from './types';
 
-export const opn = require('opn') as (
-  url: string,
-  options?: {
-    wait?: boolean;
-  },
-) => Promise<childProcess.ChildProcess>;
-
+export { opn };
```

Comment:
Since we don't add a type cast to `opn` anymore, can we get rid of this export and import from `'opn'` directly wherever we need it?

Reply:
Good idea but I'd do that later?